Client/multiplayer_sa/CMultiplayerSA_Vehicles.cpp

```diff
@@ -107,6 +114,105 @@ static void _declspec(naked) HOOK_CAEVehicleAudioEntity__Initialise()
     }
 }
 
+static void CVehicleModelInfo_Shutdown()
```

Comment:
static void  __fastcall CVehicleModelInfo_Shutdown(CVehicleModelInfoSAInterface* mi)

Reply:
did you forget to push?

Reply:
I look forward to your replies in the remaining open conversations